ocs_ci/ocs/longevity.py

```diff
@@ -514,6 +517,171 @@ def validate_obcs_in_kube_job_reached_running_state(
 
         return obc_bound_list
 
+    def create_restore_verify_snapshots(
```

Comment:
Lets move these to longevity helpers once the stage2 refactor pr is merged

Reply:
Ack